Grid/qcd/action/fermion/WilsonImpl.h

```diff
@@ -261,6 +261,20 @@ typedef WilsonImpl<vComplex,  TwoIndexAntiSymmetricRepresentation, CoeffReal > W
 typedef WilsonImpl<vComplexF, TwoIndexAntiSymmetricRepresentation, CoeffReal > WilsonTwoIndexAntiSymmetricImplF;  // Float
 typedef WilsonImpl<vComplexD, TwoIndexAntiSymmetricRepresentation, CoeffReal > WilsonTwoIndexAntiSymmetricImplD;  // Double
 
+//sp 2n
+
```

Comment:
That is a different datatype that has been introduced after I forked. Since I don't know its details and whether it works with any implementation (you can see it is not there also for some SUn implementations) I think we must not use it unless we are told to do so by someone who understand it, or unless we understand it ourselves.

Reply:
Okay.